checklists.xml

```diff
@@ -304,47 +304,87 @@
               audio-prompt="cklist-audio/flt_comp_71.opus"
               auto-adv="none"/>
       </checklist>
-      <checklist name="QUICK TURN CHECKLIST" ttsname="Quick turn checklist">
+      <checklist name="QUICK TURN CHECKLIST" ttsname="Quick turn checklist"
+                 ttscallcomplete="false" ttscallnext="false"
```

Comment:
```suggestion
                 ttscallcomplete="false" ttscallnext="true"
```

`ttscallnext="false"` was a hack in the auto-fo branch to make it work when one checklist uses TTS and the other uses recorded voice; this is not needed anymore. `ttscallnext="true"` is the normal value, otherwise there's no next checklist callout at all.